frontend/src/container/Publications.jsx

```diff
@@ -55,17 +55,21 @@ import {
   setHeadTitle,
   setFirstComment,
   setPublicationList,
+  setPublicationNextPage,
   setWorkspaceDetail,
   setWorkspaceMemberList,
   updatePublication,
-  updatePublicationList
+  updatePublicationList,
+  appendPublicationList
 } from '../action-creator.sync.js'
 
 import TabBar from '../component/TabBar/TabBar.jsx'
 import FeedItemWithPreview, { LINK_TYPE } from './FeedItemWithPreview.jsx'
 
 const wysiwygId = 'wysiwygTimelineCommentPublication'
 
+const PUBLICATION_ITEM_COUNT_PER_PAGE = 3
```

Comment:
Is it normal that there are only 3? Why don't we use NUMBER_RESULTS_BY_PAGE?

Reply:
I put 3 for testing and forgot to set it back to its proper value.

Reply:
The code now uses `NUMBER_RESULTS_BY_PAGE`.